src/applications/mhv-landing-page/constants.js

```diff
@@ -0,0 +1,91 @@
+const { entries, freeze, values } = Object;
```

Comment:
It is a bit confusing that we now have links in both constants and data files.

Reply:
I don't disagree, however I think the benefit of having all the VA.gov health tools in one file outweighs the drawbacks.